recipes-nodes/lighthouse/init

```diff
@@ -72,9 +72,11 @@ start() {
 	mount -o remount,size=90% /var/volatile
 
 	# Generate a random JWT secret and save it to the file
-	openssl rand -hex 32 | tr -d "\n" | tee $JWT_SECRET_FILE > /dev/null
-	chown ${LIGHTHOUSE_USER}:${ETH_GROUP} $JWT_SECRET_FILE
-	chmod 640 $JWT_SECRET_FILE
+    if [ ! -f $JWT_SECRET_FILE ]; then
```

Comment:
do we want to persistently store the JWT_SECRET_FILE?
if yes, then you probably want to change the path in [line 24](https://github.com/flashbots/meta-evm/pull/45/files#diff-5948139458b219e6334aca514d048377bcb3cea52b6ff2e72469375464d77190R24)

Reply:
no, persistence across reboots is not a concern